commit c3e9814926e4ae596937541b21e4370f43c48c23
Author: Roderick Schaefer <roderick@wehandle.it>
Date:   Sat May 13 10:20:51 2023 +0200

    Ghidra config patches
    
    Manually run Ghidra and toggle 'overview' on/off in the code editor to poplate an initial config,
    before applying this file.

diff --git a/tools/_code_browser.tcd b/tools/_code_browser.tcd
index 0f781ab..c510d02 100644
--- a/tools/_code_browser.tcd
+++ b/tools/_code_browser.tcd
@@ -4,7 +4,27 @@
     <SUPPORTED_DATA_TYPE CLASS_NAME="ghidra.program.model.listing.Program" />
     <ICON LOCATION="greenDragon24.png" />
     <TOOL TOOL_NAME="CodeBrowser" INSTANCE_NAME="">
-        <OPTIONS />
+        <OPTIONS>
+            <CATEGORY NAME="Listing Fields">
+                <ENUM NAME="Array Options.Array Index Format" TYPE="enum" CLASS="ghidra.app.util.viewer.field.FieldNameFieldFactory$IndexFormat" VALUE="hex" />
+                <STATE NAME="Bytes Field.Display in Upper Case" TYPE="boolean" VALUE="true" />
+                <STATE NAME="Bytes Field.Maximum Lines To Display" TYPE="int" VALUE="1" />
+                <ENUM NAME="Cursor Text Highlight.Mouse Button To Activate" TYPE="enum" CLASS="ghidra.GhidraOptions$CURSOR_MOUSE_BUTTON_NAMES" VALUE="LEFT" />
+                <STATE NAME="Format Code.Flag Function Entry" TYPE="boolean" VALUE="true" />
+                <STATE NAME="Format Code.Flag Function Exits" TYPE="boolean" VALUE="true" />
+                <STATE NAME="Labels Field.Display Function Label" TYPE="boolean" VALUE="false" />
+                <STATE NAME="Mouse.Horizontal Scrolling" TYPE="boolean" VALUE="false" />
+                <STATE NAME="Operands Field.Enable Word Wrapping" TYPE="boolean" VALUE="true" />
+                <STATE NAME="Operands Field.Markup Register Variable References" TYPE="boolean" VALUE="false" />
+                <STATE NAME="Operands Field.Maximum Lines To Display" TYPE="int" VALUE="200" />
+                <STATE NAME="XREFs Field.Maximum Number of XREFs to Display" TYPE="int" VALUE="200" />
+                <WRAPPED_OPTION NAME="Array Options.Array Display Options" CLASS="ghidra.framework.options.WrappedCustomOption">
+                    <STATE NAME="CUSTOM OPTION CLASS" TYPE="string" VALUE="ghidra.app.util.viewer.field.ArrayElementWrappedOption" />
+                    <STATE NAME="elementsPerLine" TYPE="int" VALUE="15" />
+                    <STATE NAME="showMultiArrayElementsPerLine" TYPE="boolean" VALUE="true" />
+                </WRAPPED_OPTION>
+            </CATEGORY>
+        </OPTIONS>
         <PACKAGE NAME="Ghidra Core">
             <INCLUDE CLASS="ghidra.app.plugin.core.editor.TextEditorManagerPlugin" />
             <INCLUDE CLASS="ghidra.app.plugin.core.interpreter.InterpreterPanelPlugin" />
@@ -26,7 +46,9 @@
             <STATE NAME="word" TYPE="string" VALUE="/" />
         </PLUGIN_STATE>
         <PLUGIN_STATE CLASS="ghidra.app.plugin.core.overview.OverviewColorPlugin">
-            <ARRAY NAME="ActiveServices" TYPE="string" />
+            <ARRAY NAME="ActiveServices" TYPE="string">
+                <A VALUE="Overview" />
+            </ARRAY>
         </PLUGIN_STATE>
         <PLUGIN_STATE CLASS="ghidra.app.plugin.core.symboltree.SymbolTreePlugin">
             <STATE NAME="GO_TO_TOGGLE_STATE" TYPE="boolean" VALUE="false" />
